iml-gui/crate/src/breakpoints.rs

```diff
@@ -6,40 +6,35 @@
 
 use seed::window;
 
-pub(crate) const SM: f64 = 569.;
-pub(crate) const MD: f64 = 769.;
-pub(crate) const LG: f64 = 1025.;
-pub(crate) const XL: f64 = 1701.;
-
 #[derive(Clone, Copy, Debug, Eq, PartialEq, Ord, PartialOrd)]
 pub(crate) enum Size {
-    XS,
-    SM,
-    MD,
-    LG,
-    XL,
+    XS = 0,
+    SM = 569,
+    MD = 769,
+    LG = 1025,
+    XL = 1701,
 }
 
-fn inner_width() -> f64 {
+fn inner_width() -> u64 {
     window()
         .inner_width()
         .expect("Could not get inner_width")
         .as_f64()
-        .expect("Could not parse inner_width to f64")
+        .expect("Could not parse inner_width to f64") as u64
 }
 
 /// Returns *maximum* matching breakpoint based on `window.inner_width`.
 /// This models the behavior of breakpoints in tailwind: https://tailwindcss.com/docs/responsive-design/
 pub(crate) fn size() -> Size {
     let w = inner_width();
 
-    if w >= XL {
+    if w >= Size::XL as u64 {
```

Comment:
Doesn't it work without type casting? O_o

Reply:
Nope

Reply:
Which is what I was referring to earlier. However, the compiler / rust-analyzer will tell you if you don't cast, so I think it's ok.

Reply:
I feel something like regret :)

Reply:
Oh, maybe use `#[repr]` for the enum?

Reply:
Nope, still same issue with `repr` attribute.